packages/create-app/template-vanilla-ts/src/main.ts

```diff
@@ -0,0 +1,8 @@
+import './style.css'
+
+const app = document.querySelector('#app') as HTMLDivElement
+
+app.innerHTML = `
+  <h1>Hello Vite!</h1>
+  <a href="https://vitejs.dev/guide/features.html" target="_blank">Documentation</a>
+`
```

Comment:
```suggestion
const app = document.querySelector('#app')

if (app) {
  app.innerHTML = `
  <h1>Hello Vite!</h1>
  <a href="https://vitejs.dev/guide/features.html" target="_blank">Documentation</a>
`
}
```

This type assertion (`as HTMLDivElement`) "hides" an `Object is possibly 'null'` compile-time error, without actually preventing an `Uncaught TypeError: Cannot set property 'innerHTML' of null` runtime error from being thrown, in the event that `document.querySelector('#app')` fails to return an element.

(bit of a nitpick really, and I could see others deciding this is a great scenario to use `as`, since it's unlikely you'd run into this error in practice)

Reply:
I kinda think it's better to let it throws when the `app` is undefined instead of failing silently.

Changed to 

```ts
const app = document.querySelector<HTMLDivElement>('#app')!
```